protocol-units/bridge/service/src/rest.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::Error;
+use aptos_api::Context;
+use bridge_config::common::movement::MovementConfig;
+use futures::prelude::*;
+use poem::{
+	get, handler, listener::TcpListener, middleware::Tracing, EndpointExt, IntoResponse, Response,
+	Route, Server,
+};
+use std::future::Future;
+use std::sync::Arc;
+use tracing::info;
+
+#[derive(Debug)]
+pub struct BridgeRest {
```

Comment:
We may want to generalize this pattern going forward. With tonic, you can directly do GRPC over HTTP1. But, it's still a little awkward for nullaries, like health checks, because a message type is still expected. 

gRPC on the other hand has a health standard that we might want to adopt: https://github.com/grpc/grpc/blob/master/doc/health-checking.md

Reply:
I'll use this and add in a test, I think REST health is still useful for quick checking also. So health on both.